outport/outport.go

```diff
@@ -25,73 +25,108 @@ func NewOutport() *outport {
 }
 
 // SaveBlock will save block for every driver
-func (o *outport) SaveBlock(args *indexer.ArgsSaveBlockData) {
+func (o *outport) SaveBlock(args *indexer.ArgsSaveBlockData) error {
 	o.mutex.RLock()
 	defer o.mutex.RUnlock()
 
 	for _, driver := range o.drivers {
-		driver.SaveBlock(args)
+		err := driver.SaveBlock(args)
```

Comment:
so in case we have multiple drivers and one of them has an issue we no longer call SaveBlock for the rest?

maybe call for all SaveBlock, and if at least one of them returned an error log all of them and report the last error?

could be done for the rest of the methods in outport

Reply:
sure, we can do that